From: Revanth Rajashekar <revanth.rajashekar@intel.com>
Date: Tue, 20 Aug 2019 09:30:51 -0600
Subject: [PATCH] block: sed-opal: Removed duplicate OPAL_METHOD_LENGTH
 definition
References: bsc#1156419,jsc#SLE-8281
Git-commit: 238bdcdf5d0a087f8930c82b7c7cc142ca9399ce
Patch-mainline: v5.4-rc1

The original commit adding the sed-opal library by mistake added two
definitions of OPAL_METHOD_LENGTH, remove one of them.

Signed-off-by: Revanth Rajashekar <revanth.rajashekar@intel.com>
Reviewed-by: Scott Bauer <sbauer@plzdonthack.me>
Reviewed-by: Jon Derrick <jonathan.derrick@intel.com>
Signed-off-by: Jens Axboe <axboe@kernel.dk>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 block/opal_proto.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/block/opal_proto.h b/block/opal_proto.h
index 562b78f40824..5532412d567c 100644
--- a/block/opal_proto.h
+++ b/block/opal_proto.h
@@ -119,8 +119,6 @@ enum opal_uid {
 	OPAL_UID_HEXFF,
 };
 
-#define OPAL_METHOD_LENGTH 8
-
 /* Enum for indexing the OPALMETHOD array */
 enum opal_method {
 	OPAL_PROPERTIES,
-- 
2.16.4

